src/ManagerInterface.h

```diff
@@ -27,19 +27,18 @@ namespace ClassProject {
 
         virtual BDD_ID topVar(BDD_ID f) = 0;
 
-        virtual std::string getTopVarName(const BDD_ID &root) = 0;
+        virtual BDD_ID ite(BDD_ID i, BDD_ID t, BDD_ID e) = 0;
 
-        //virtual BDD_ID and2(const std::string &a, const std::string &b) = 0;
+        virtual BDD_ID and2(std::string &a, std::string &b) = 0;
```

Comment:
I think this function should be:
```C++
virtual BDD_ID and2(BDD_ID a, BDD_ID b) = 0;
```
not
```C++
virtual BDD_ID and2(std::string &a, std::string &b) = 0;
```
Shall we delete it from this pull request and add in separate branch, with or2, not, xor etc? @Adhaywe @emanuelacm

Reply:
I make that change because in the pdf when we a*b is called then this format "and2(a, b)" is used and it then calls the ite function. I thought we are calling a and b variables not with their IDs but rather with their actual names. But if it can implemented in a better  way let me know.